python/ray/autoscaler/_private/autoscaler.py

```diff
@@ -238,30 +242,55 @@ def read_fn():
         self.disable_node_updaters = self.config["provider"].get(
             "disable_node_updaters", False
         )
+        if self.disable_node_updaters:
```

Comment:
As a general comment it's a good practice to use the positive style for flags. People often get confused when they see "disableXXX == False" given that it is a double negative. Better to say "enableXXX". I realize you may not be able to make this change here - that is fine.

Reply:
Renamed it "Foreground node launch."